client/pages/clubs/[id].tsx

```diff
@@ -54,6 +55,7 @@ export default function ClubDetail() {
 
 const BackArrow = styled.img`
   margin-top: 64px;
+  cursor: pointer;
```

Comment:
I'm quite unfamiliar with HTML/CSS but this kind of feels like a hack to make the image act like a link. Is it possible to do something like `<a><BackArrow ... /></a>` to make it automatically interpret the picture as a link?

Reply:
I believe there's a distinction being made in the webdev community between external links and in-app navigational components.

External links are actually links that link to websites outside of the current app. We should use `<a>` tags to give them the correct semantic meaning.

Internal navigational components like buttons to move forwards and backwards throughout the SPA arguably don't need `<a>` wrapped around itself. Thoughts?

Reply:
Something like this? https://stackoverflow.com/questions/21715162/when-should-i-use-a-button-button-or-a-link-a-in-html

Makes sense, let's keep it the way it is then